test/quadquizaminos/contests/contest/rsvp_test.exs

```diff
@@ -5,11 +5,29 @@ defmodule Quadquizaminos.ContestsTest.RSVP do
   alias Quadquizaminos.Contests
   alias Quadquizaminos.Contests.RSVP
 
-  test "create_rsvp/2 creates a new rsvp on db" do
-    {:ok, current_user} = Accounts.create_user(%User{}, %{name: "ha", uid: "34"})
-    {:ok, contest} = Contests.create_contest(%{name: "test contest"})
-    attrs = %{"contest_id" => contest.id}
+  describe "RSVP" do
+    setup do
+      {:ok, current_user} = Accounts.create_user(%User{}, %{name: "ha", uid: "34"})
+      {:ok, contest} = Contests.create_contest(%{name: "test contest"})
+      attrs = %{"contest_id" => contest.id}
 
-    {:ok, %RSVP{}} = Contests.create_rsvp(attrs, current_user)
+      [attrs: attrs, current_user: current_user, contest: contest]
+    end
+
+    test "create_rsvp/2 creates a new rsvp on db", %{attrs: attrs, current_user: current_user} do
+      {:ok, %RSVP{}} = Contests.create_rsvp(attrs, current_user)
```

Comment:
Don't we need to assert that a new rsvp has been created?

Reply:
With `{: ok, rsvp}` also indicates that rsvp was successfully inserted into the db.

Reply:
But what happens when the rsvp is not successfully written to the DB? I find assertions make it easy to outline what was expected versus what was returned, saving you debugging time